config/default/common/config/metadata/layers/multi-mission/hls/HLS_L30_Nadir_BRDF_Adjusted_Reflectance.md

```diff
@@ -1,5 +1,5 @@
-The Reflectance (Nadir BRDF Adjusted) imagery layer from Landsat 8/OLI is a true-color or natural color image because this combination of wavelengths is similar to what the human eye would see. It consists of natural-looking images of land surface, oceanic and atmospheric features. The Reflectance (Nadir BRDF Adjusted) imagery layer from Landsat 8/OLI provides 30m Nadir Bidirectional Reflectance Distribution Function (BRDF)-Adjusted Reflectance (NBAR) and is derived from the joint NASA/USGS Landsat 8 Operational Land Imager (OLI) data product through the Harmonized Landsat and Sentinel-2 (HLS) project.
+The Reflectance (Nadir BRDF Adjusted) imagery layer from Landsat 8 and 9/OLI is a true-color or natural color image because this combination of wavelengths is similar to what the human eye would see. It consists of natural-looking images of land surface, oceanic and atmospheric features. The Reflectance (Nadir BRDF Adjusted) imagery layer from Landsat 8 and 9/OLI provides 30m Nadir Bidirectional Reflectance Distribution Function (BRDF)-Adjusted Reflectance (NBAR) and is derived from a combination the joint NASA/USGS Landsat 8 and Landsat 9 Operational Land Imager (OLI) data products through the Harmonized Landsat and Sentinel-2 (HLS) project.
```

Comment:
`a combination the joint NASA/USGS` => `a combination of the joint NASA/USGS`

Reply:
good eyes!